package.json

```diff
@@ -13,6 +13,7 @@
     "reshape": "^0.4.0",
     "reshape-loader": "^0.4.0",
     "rooftop-client": "^0.2.1",
+    "spike-util": "^0.4.0",
```

Comment:
Hmm.. I had set it to ^0.4.0 since that version is also referenced by spike-core 0.13.3, which in turn is in spike-rooftop's dev-dependencies. But indeed - changing it to 1.0.0 seems to get me one step further. There's a new error now.

I'll try and set up my local project for testing, so I can fix it once and for all before committing and having Travis do the work :p  Thanks so far for your help.